server/api/handlers/user/profile.py

```
@@ -57,18 +59,34 @@ def upload_image_to_cloudinary(image):
     except Exception as e:
         logging.error(f"Error uploading image: {str(e)}")
         return None
+    
+def update_banner(username):
+    data = request.form
```

Ruff

Local variable `data` is assigned to but never used

Remove assignment to unused variable `data`

(F841)